draft-ietf-rtcweb-jsep.xml

```diff
@@ -5680,6 +5582,167 @@ a=rtcp-fb:100 nack pli
         <xref target="I-D.ietf-mmusic-dtls-sdp" />Section 4</c>
       </texttable>
     </section>
+
+    <section title="Appendix B" anchor="sec.appendix-b">
+      <t>The following text is meant to completely replace section
+      "Associating RTP/RTCP Streams With Correct SDP Media
+      Description" of mmusic-sdp-bundle-negotiation. </t>
+      
+        <t>
+          As described in RFC3550, RTP/RTCP packets
+          are associated with RTP streams as defined in RFC7656. Each RTP stream is identified by an SSRC value, and each
+          RTP/RTCP packet carries an SSRC value that is used to associate the packet with the correct
+          RTP stream (an RTCP packet can carry multiple SSRC values, and might therefore be associated
+          with multiple RTP streams).
+        </t>
+        <t>
+          In order to be able to process received RTP/RTCP packets correctly it must be possible
+          to associate an RTP stream with the correct "m=" line, as the "m=" line and SDP attributes
+          associated with the "m=" line contain information needed to process the packets.
+        </t>
+        <t>
+          As all RTP streams associated with a BUNDLE group are using the same
+          address:port combination for sending and receiving RTP/RTCP packets, the
+          local address:port combination cannot be used to associate an RTP stream
+          with the correct "m=" line. In addition, multiple RTP streams might be
+          associated with the same "m=" line.
+        </t>
+        <t>
+          An offerer and answerer can inform each other which SSRC values they will use
+          for an RTP stream by using the SDP 'ssrc' attribute <xref format="default" pageno="false"
+          target="RFC5576"/>. However, an offerer will not know which SSRC values the
+          answerer will use until the offerer has received the answer providing that information.
+          Due to this, before the offerer has received the answer, the offerer will not be
+          able to associate an RTP stream with the correct "m=" line using the SSRC value associated
+          with the RTP stream. In addition, the offerer and answerer may start using new SSRC
+          values mid-session, without informing each other using the SDP 'ssrc' attribute.
+        </t>
+        <t>
+          In order for an offerer and answerer to always be able to associate an RTP stream
+          with the correct "m=" line, the offerer and answerer using the BUNDLE extension MUST
+          support the mechanism defined in TODO, <!--<xref format="default" pageno="false" target="sec-receiver-id"/> -->,
+          where the offerer and answerer insert the identification-tag (provided by the remote peer)
```

Comment:
this "provided by the remote peer" comes at an awkward time here; might be less confusing after "associated with an "m=" line"